File: 900/2102B/main.cpp
#include <bits/stdc++.h>
using namespace std;

int main() 
{
    int t;
    cin >> t;

    while (t--) 
    {
        int n;
        cin >> n;

        int first;
        cin >> first;
        first = abs(first);

        int cnt = 0;
        for (int i = 1; i < n; i++) 
        {
            int x;
            cin >> x;

            if (abs(x) < first)
                cnt++;
        }

        cout << (cnt <= n / 2 ? "YES" : "NO") << "\n";
    }

    return 0;
}

File: 900/2102B/main.java
import java.util.*;
public class main
{
    public static void main(String args[])
    {
        Scanner sc = new Scanner(System.in);
        int t = sc.nextInt();
        
        while(t-- > 0)
        {
            int n = sc.nextInt();
            int first = Math.abs(sc.nextInt());
            int cnt = 0;
 
            for(int i=1;i<n;i++)
            {
                if(Math.abs(sc.nextInt()) < first)
                    cnt++;
            }
 
            System.out.println(cnt <= n/2 ? "YES" : "NO");
        }
    }
}